Index: beryllium.pro
===================================================================
--- beryllium.pro	(revision 21448)
+++ beryllium.pro	(working copy)
@@ -19,7 +19,8 @@
 
 dicom {
     DEFINES += WITH_DICOM
-    LIBS += -ldcmnet -lwrap -li2d -ldcmdata -ldcmtls -loflog -lofstd -lssl
+#    LIBS += -ldcmnet -lwrap -li2d -ldcmdata -ldcmtls -loflog -lofstd -lssl
+     LIBS += -L/usr/lib/dcmtk -ldcmnet -llibi2d -ldcmdata -ldcmtls -loflog -lofstd -lssl
     SOURCES += worklist.cpp startstudydialog.cpp dcmclient.cpp detailsdialog.cpp
     HEADERS += worklist.h startstudydialog.h dcmclient.h detailsdialog.h
 }
